### PR TITLE
bavarder: init at 0.1.0

### PR DESCRIPTION
**Pull request by 0xMRTT**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/baichat-py/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    hash = "sha256-g03jyNVEyXJPCJ17gFG6xXh9UZt2nOyskFQFnvVZsPs=";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+  ];
+
+  # No tests implemented.
+  doCheck = false;
+}
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/baichat-py/default.nix:24]: Missing meta and pythonImportsCheck

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by Zahrun** [comment]
Thanks @0xMRTT 
It seems the tag 0.1.0 was deleted, maybe use 0.1.3
Would be nice to have Imaginer too https://github.com/ImaginerApp/Imaginer

**Comment by 0xMRTT** [comment]
See #230079

**Comment by Tommimon** [comment]
What is the status of this? What is preventing the merge? The corresponding request for imaginer was merged in one day.

File: pkgs/development/python-modules/baichat-py/default.nix (hunk 1 above)
Comment (Zahrun) [reply pkgs/development/python-modules/baichat-py/default.nix:24]: @0xMRTT can you please check?

**Comment by 0xMRTT** [comment]
#290501 is going to replace it I guess

**Pull request closed without merging**